.eslintrc.js

```diff
@@ -6,9 +6,9 @@ module.exports = {
     'plugin:@typescript-eslint/recommended',
     'prettier/@typescript-eslint',
     'plugin:prettier/recommended',
-    'prettier/react',
+    'prettier/react'
   ],
-  plugins: ['react', '@typescript-eslint', 'prettier'],
+  plugins: ['react', '@typescript-eslint','prettier'],
```

Comment:
inconsistent spacing.. prettier would catch this  😛

Reply:
✔️ - double checked prettier still in use, re-ran prettier on eslintrc.js